### DISCUSSION TITLE
How to pass options to `part_graph` such as `ufactor`?

### DISCUSSION THREAD

**Discussion by alexbovet** [inducer/pymetis#82] [Q&A] [answered]
I see in the code that one can pass an `options` parameter to `part_graph` and I also saw that there is an `Options` class that I think should be used for that. However, this is not in the documentation.
I would like to pass a value for `ufactor`, and I guess this can be done though these options.
Can you tell me how the options should be used?
Thanks!

**Comment by inducer** [comment, marked as answer]
The [relevant test](https://github.com/inducer/pymetis/blob/fd878a4bef732e2236462b2fe4809f80c7fe3ab7/test/test_metis.py#L130-L171) may serve as a usable example.

**Reply by inducer** [reply]
If you'd like to contribute a documentation improvement, a PR would be very welcome.

**Comment by alexbovet** [comment]
thanks!
